airbyte-db/db-lib/build.gradle

```diff
@@ -2,6 +2,10 @@ plugins {
     id 'java-library'
 }
 
+configurations {
```

Comment:
@pedroslopez before you merge in, can you add a comment here explaining why we do this? Thanks.